src/attribute_type.rs

```diff
@@ -2053,9 +2053,10 @@ fn parse_image_attr_xyrange(
             Ok(SdpAttributeImageAttrXYRange::DiscreteValues(values))
         }
     } else {
-        Ok(SdpAttributeImageAttrXYRange::DiscreteValues(vec![
-            to_parse.parse::<u32>()?
-        ]))
+        // Using the vec! macro directly leaks memory
```

Comment:
I would like to see some type of reference to why this happens, rust docs, a bug against rustc, etc (or remove the comment).

Reply:
Since I removed that new code and reverted back to the old code I also removed the comment.
But I have to admit that I don't fully grog why this is leaking without the length parameter. I started asking around.